hikari/channels.py

```diff
@@ -974,6 +974,19 @@ def shard_id(self) -> typing.Optional[int]:
 
         return None
 
+    def get_guild(self) -> typing.Optional[guilds.GatewayGuild]:
+        """Return the guild linked to this channel.
+
+        Returns
+        -------
+        typing.Optional[hikari.guilds.Guild]
+            `builtins.None` if the guild is not cached.
```

Comment:
Since this is a description of the return, the following might be a bit more fitting
```suggestion
            The linked guild object, or `builtins.None` if it's not cached.
```

Reply:
I used the same description in my `hikari.Member.get_guild` PR too.
Might want to change that too then?

Reply:
You can just change that in this pr if you want to

Reply:
Done.